docs/modules/maintain-cluster/pages/rolling-upgrades.adoc

```diff
@@ -48,8 +60,23 @@ As a result, you can use rolling upgrades to upgrade clusters to the following m
 +
 This also includes the ability of rolling upgrade from Hazelcast IMDG 4.2 to Platform 5.0, since IMDG is part of the Platform and 5.0 is considered to be the next minor version after IMDG 4.2.
 
+- Releases from 5.2 to 5.4, directly to the 5.5 LTS release. For further information on additional compatibilities for direct-to-LTS rolling upgrades, see xref:migrate:lts.adoc[].
```

Comment:
Upon first reading it was not clear to me that 5.3 is also supported. My suggestion is : Releases 5.2 , 5.3 and 5.4 can be upgraded directly to the 5.5 LTS release.